workspaces/tests/batch_tx.rs

```diff
@@ -0,0 +1,41 @@
+use serde_json::json;
+use test_log::test;
+use workspaces::prelude::*;
+use workspaces::CallArgs;
+
+#[test(tokio::test)]
+async fn test_batch_tx() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox();
+    let contract = worker
+        .dev_deploy(include_bytes!("../../examples/res/status_message.wasm"))
+        .await?;
+
+    // Batch transaction with two `call`s into `set_status`. The second one
+    // should override the first one.
+    contract
+        .batch_tx(&worker)
+        .call(
+            CallArgs::new("set_status")
+                .args_json(json!({
+                    "message": "hello_world",
+                }))?
+                .deposit(0),
+        )
+        .call(CallArgs::new("set_status").args_json(json!({
+            "message": "world_hello",
+        }))?)
```

Comment:
The stylistic discrepancy between these two calls looks pretty weird, but I assume this is because of rustfmt...

In general I am not a big fan of how this "reads", but at the same time I can't come up with anything better right now.